build/ci/official.yml

```diff
@@ -82,12 +81,10 @@ steps:
   displayName: Archive symbols to VSTS
   inputs:
     SymbolsFolder: $(Build.ArtifactStagingDirectory)/symbols
-    SearchPattern: 
-      '**/*.pdb
-      **/*.dll
-      **/*.exe'
+    SearchPattern: '**/*.pdb'
```

Comment:
Don't we need to archive the binaries as well?

Reply:
Good catch